float32[] position
float32[] velocity
float32[] torque
int32 gripperReady
